arc-manager.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ids": {
+    "ManagerHost": 2,
+    "LoadBalancer": 1
+  },
+  "models": {
+    "ManagerHost": {
+      "73ec9e68-5709-4172-821d-02a9d6d948f0": "{\"id\":\"73ec9e68-5709-4172-821d-02a9d6d948f0\",\"host\":\"localhost\",\"port\":8701,\"actions\":[\"delete\",\"edit\",\"env-set\",\"env-get\",\"stop\",\"restart\",\"cluster-restart\",\"license-push\"],\"protocol\":\"http\",\"created\":1440105768114,\"error\":null,\"errorType\":null,\"app\":{\"name\":\"dbload-example-app2\",\"version\":null,\"port\":3001},\"processes\":{\"pids\":[{\"parentPid\":2212,\"pid\":2213,\"workerId\":1,\"listeningSockets\":[{\"address\":\"0.0.0.0\",\"port\":3001,\"addressType\":4}],\"isTrackingObjects\":false,\"isProfiling\":false,\"isTracing\":true,\"watchdogTimeout\":0,\"watchdogStallout\":0,\"isSnapshotting\":false,\"startTime\":\"2015-08-20T23:30:57.540Z\",\"stopTime\":null,\"stopReason\":\"\",\"id\":4,\"serviceInstanceId\":1},{\"parentPid\":2201,\"pid\":2212,\"workerId\":0,\"listeningSockets\":[],\"isTrackingObjects\":false,\"isProfiling\":false,\"isTracing\":false,\"watchdogTimeout\":0,\"watchdogStallout\":0,\"isSnapshotting\":false,\"startTime\":\"2015-08-20T23:30:57.233Z\",\"stopTime\":null,\"stopReason\":\"\",\"id\":5,\"serviceInstanceId\":1}]}}"
+    },
+    "LoadBalancer": {}
+  }
+}
```

Comment:
this shouldn't be committed, I don't think, its your private arc file. arc-manager.json should be in the .gitignore